src/shadowbox/docker/Dockerfile

```diff
@@ -28,6 +28,9 @@ LABEL shadowbox.github.release="${GITHUB_RELEASE}"
 # We use curl to detect the server's public IP.
 RUN apk add --no-cache curl
```

Comment:
Put all the packages in the same apk add command, do reduce the number of layers

Reply:
Fixed in commit b5b930c